lib/bloc/bus_bloc.dart

```diff
@@ -17,6 +18,8 @@ class BusBloc extends Cubit<BusState> {
   final formKey = GlobalKey<FormState>();
   List<RouteData> routesData = [];
   List<VehiclesData> vehiclesData = [];
+  final HiveService hiveService = HiveService();
+  List list = [];
```

Comment:
Change variable name list to searchRecords

Reply:
Done